homeassistant/components/efergy/sensor.py

```diff
@@ -32,11 +42,26 @@
 DEFAULT_UTC_OFFSET = "0"
 
 SENSOR_TYPES = {
-    CONF_INSTANT: ["Energy Usage", POWER_WATT],
-    CONF_AMOUNT: ["Energy Consumed", ENERGY_KILO_WATT_HOUR],
-    CONF_BUDGET: ["Energy Budget", None],
-    CONF_COST: ["Energy Cost", None],
-    CONF_CURRENT_VALUES: ["Per-Device Usage", POWER_WATT],
+    CONF_INSTANT: {
```

Comment:
If you refactor this into a dictionary, please use a `SensorEntityDescription` instead.